bin/doAnalysis.cc

```diff
@@ -824,33 +880,33 @@ int main(int argc, char** argv)
     std::cout<<setprecision(2);
     std::cout<<right;
     std::cout<< "Timing summary"<<std::endl;
-    std::cout<< "Evt     Hits       MD   Segments Triplets  Pixels  pTracklet  Tracklet  Tracks    Total  Total(no load)"<<std::endl;
+    std::cout<< "Evt     Hits       MD   Segments Triplets  pTracklet  Tracklet  Tracks    Total  Total(no load)"<<std::endl;
     float avg_hit = 0.;
     float avg_md = 0.;
     float avg_seg = 0.;
     float avg_trip = 0.;
-    float avg_pix = 0.;
+    //float avg_pix = 0.;
     float avg_plet = 0.;
     float avg_tlet = 0.;
     float avg_track = 0.;
     float avg_tot = 0.;
     float avg_noload = 0.;
     for(int ev=0;ev<ana.n_events;ev++){
-        float total =event_times[ev][0]+event_times[ev][1] +event_times[ev][2]+event_times[ev][3]+event_times[ev][4]+ event_times[ev][5]+event_times[ev][6]+event_times[ev][7];
-        float no_load =event_times[ev][1] +event_times[ev][2]+event_times[ev][3]+ event_times[ev][5]+event_times[ev][6]+event_times[ev][7];
+        float total =event_times[ev][0]+event_times[ev][1] +event_times[ev][2]+event_times[ev][3]+event_times[ev][4]+ event_times[ev][5]+event_times[ev][6];//+event_times[ev][7];
+        float no_load =event_times[ev][1] +event_times[ev][2]+event_times[ev][3]+ event_times[ev][5]+event_times[ev][6];//+event_times[ev][7];
```

Comment:
@tresreid 
I think that your comment was referring to this line https://github.com/SegmentLinking/TrackLooper/pull/15#issuecomment-748220363

you can also click on the "+/-" box above and make a suggestion
```suggestion
        float no_load =event_times[ev][1] +event_times[ev][2]+event_times[ev][3]+event_times[ev][4]+ event_times[ev][5]+event_times[ev][6];//+event_times[ev][7];
```